shared/chat/conversation/messages/message-popup/payment/index.stories.js

```diff
@@ -0,0 +1,120 @@
+// @flow
+import * as React from 'react'
+import * as Sb from '../../../../../stories/storybook'
+import * as S from '../../../../../styles'
+import {isMobile} from '../../../../../constants/platform'
+import PaymentPopup from '.'
+
+const sendIcon = isMobile ? 'icon-fancy-stellar-sending-mobile' : 'icon-fancy-stellar-sending-desktop'
+const receiveIcon = isMobile ? 'icon-fancy-stellar-receiving-mobile' : 'icon-fancy-stellar-receiving-desktop'
+
+const commonProps = {
+  attachTo: null,
+  onCancel: null,
+  onHidden: Sb.action('onHidden'),
+  position: 'top right',
+  senderDeviceName: 'iPhone 6',
+  timestamp: 'Yesterday 8:11 PM',
+  visible: true,
+}
+
+const onCancel = Sb.action('onCancel')
+
+const theyRequestProps = {
+  ...commonProps,
+  amountNominal: '$34',
+  balanceChange: '',
+  balanceChangeColor: '',
+  bottomLine: '',
+  icon: receiveIcon,
+  sender: 'kamel',
+  topLine: 'requested lumens worth',
+  txVerb: 'requested',
+}
+
+const youReceiveProps = {
+  ...commonProps,
+  amountNominal: '$1',
+  balanceChange: '+5.0200803 XLM',
+  balanceChangeColor: S.globalColors.green2,
+  bottomLine: '',
+  icon: receiveIcon,
+  sender: 'kamel',
+  topLine: 'you received lumens worth',
+  txVerb: 'sent',
+}
+
+const youRequestProps = {
+  ...commonProps,
+  amountNominal: '$34',
+  balanceChange: '',
+  balanceChangeColor: '',
+  bottomLine: '',
+  icon: receiveIcon,
+  onCancel,
+  sender: 'cecileb',
+  topLine: 'you requested lumens worth',
+  txVerb: 'requested',
+}
+
+const youSendProps = {
+  ...commonProps,
+  amountNominal: '$1',
+  balanceChange: '-170.6827309 XLM',
+  balanceChangeColor: S.globalColors.red,
+  bottomLine: '',
+  icon: sendIcon,
+  sender: 'cecileb',
+  topLine: 'you sent lumens worth',
+  txVerb: 'sent',
+}
+
+const youRequestBTCProps = {
+  ...commonProps,
+  amountNominal: '3 BTC',
+  balanceChange: '',
+  balanceChangeColor: '',
+  bottomLine: 'stronghold.com',
+  icon: receiveIcon,
+  onCancel,
+  sender: 'cecileb',
+  topLine: 'you requested',
+  txVerb: 'requested',
+}
+
+const youReceiveBTCProps = {
+  ...commonProps,
+  amountNominal: '1 BTC',
+  balanceChange: '+1 BTC',
+  balanceChangeColor: S.globalColors.green2,
+  bottomLine: 'stronghold.com',
+  icon: receiveIcon,
+  sender: 'kamel',
+  topLine: 'you received',
+  txVerb: 'sent',
+}
+
+const youSendBTCProps = {
+  ...commonProps,
+  amountNominal: '1 BTC',
+  balanceChange: '-1 BTC',
+  balanceChangeColor: S.globalColors.red,
+  bottomLine: 'stronghold.com',
+  icon: sendIcon,
+  sender: 'cecileb',
+  topLine: 'you sent',
+  txVerb: 'sent',
+}
+
+const load = () => {
+  Sb.storiesOf('Chat/Conversation/Message popup/Payments', module)
```

Comment:
Could you maybe add a container provider with some top and left margin so we can see the overhanging part of the popup?
<img width="471" alt="screen shot 2018-08-15 at 1 20 54 pm" src="https://user-images.githubusercontent.com/5677971/44162351-1309a700-a08e-11e8-9e41-c5d74c0a91fc.png">

Reply:
The decorator doesn't wrap the `div#modal-root` this ends up in... maybe it should? I'll see if that's hard

Reply:
I added a custom wrapper instead